test/Desalt.Core.Tests/Translation/TranslatorTests.Declarations.cs

```diff
@@ -333,6 +333,47 @@ class A {
 ");
         }
 
+        [Test]
+        [Ignore("Not yet implemented")]
+        public async Task Translate_should_declare_global_functions_for_GlobalMethods_classes_in_declarations()
+        {
+            await AssertTranslation(
+                @"
+namespace N {
+
+    [GlobalMethods]
+    static class A
+    {
+        public static void Method1() { }
+
+        [PreserveName]
+        static void Method2() {}
+
+        [PreserveCase]
+        static void Method3() {}
+
+        [ScriptName(""Renamed"")]
+        static void Method4() { }
+
+        [InlineCode(""FooModule.foo({a})"")]
+        static void Method5(int a) { }
+
+        [ScriptSkip]
+        static void Method6(int a) { }
+    }
```

Comment:
I don't remember if [GlobalMethods] is strictly for method definitions. Are fields covered with this too? I think probably not, but I haven't verified.

Reply:
Yes, they are.  It's in the changelog, and I've verified that Saltarelle also translates fields.